app/planet_routes.py

```diff
@@ -0,0 +1,52 @@
+from flask import Blueprint, jsonify, abort, make_response
+
+class Planet:
+	def __init__(self, id, name, description, no_life=True):
+		self.id = id
+		self.name = name
+		self.description = description
+		self.no_life = no_life 
+
+	def to_dict(self):
+		return dict(
+			id=self.id,
+			name=self.name,
+			description=self.description,
+			no_life=self.no_life)
+
+
+planets = [
+    Planet(1, "Methuselah", "oldest exoplanet"),
+    Planet(2, "Epsilon Eridani b", "closest exoplanet, has 2 astroid belts!!"),
+    Planet(3, "Aquarii A", "the most suns!"),
+    Planet(4, "Gliese 876 b", "gas giant, big, icy moons, water might be found", False)
```

Comment:
Read it here first: it's False that there's no life on Gliese 876b!

Reply:
Although we'll be getting rid of this collection soon, consider using named arguments here to help this be a little more self-documenting about what the various values are. Id and name are pretty clear, but the other params, especially the bare boolean, aren't as obvious. Having those as named arguments would make it explicit.